File: perl/src/tests/t_get_realtime.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

($s,$n) = Atrshmlog::get_realtime();

print "get realtime : $s : $n : \n";

print " \n";

exit (0);

# end of main

File: python/src/tests/t_get_realtime.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

(s,n) = atrshmlog.get_realtime()

print('get realtime : ' + str(s) + ' : ' + str(n) + ' : ')

print (' ')

exit(0)

# end of test

